### ISSUE TITLE
Update Homebrew Cask to v0.8.1

### ISSUE THREAD

**Issue by mmcdole** [oclint/oclint#231]
Currently 'brew cask install oclint' installed version 0.7.  Can we have the cask updated to the latest stable version?

**Comment by HeEAaD** [comment]
I updated the cask formula to 0.8.1 last week.

Run `brew cask update` before  `brew cask install oclint`.

**Comment by DennisOSRM** [comment]
0.8.1 is what gets installed. We should be good to close here.

```bash
$ oclint --version
LLVM (http://llvm.org/):
  LLVM version 3.5.0svn
  Optimized build.
  Built Oct 19 2014 (09:55:19).
  Default target: x86_64-apple-darwin14.3.0
  Host CPU: x86-64

OCLint (http://oclint.org/):
  OCLint version 0.8.1.
  Built Oct 19 2014 (10:24:44).
```

**Comment by ryuichis** [comment]
I am closing this, but would like to see if anyone would like to update the homebrew crew with 0.10.1 release. Thanks for the contributions.

**Comment by ryuichis** [comment]
Looks like I commented too fast, and then realized that it has been updated already: https://github.com/caskroom/homebrew-cask/blob/ad51d2492ef02fd834c5c41251c576944eb2493a/Casks/oclint.rb
